cas/client.h

```diff
@@ -475,11 +479,11 @@ M0_INTERNAL int m0_cas_get(struct m0_cas_req      *req,
 
 /**
  * A version-aware version of m0_cas_get.
- * It returns the value associated with the given key only this value
- * does not have a tombstone set (i.e., it was not removed).
- * Ssee ::COF_VERSIONED.
+ * It returns the values associated with the given keys only if this values
+ * do not have tombstones set (i.e., they have not been removed).
+ * See ::COF_VERSIONED.
  *
- * TODO: This function will be disolved once ::m0_cas_get gets an extra
+ * TODO: This function may be disolved if ::m0_cas_get gets an extra
```

Comment:
RE: m0_cas_versioned_get()

maybe having just m0_cas_get(req) with preceding req->flags |= VERSIONED will be sufficient?

Reply:
Nope. m0_cas_req has no flags. Unfortunately, there is no field to set before or after m0_cas_get call.